sigar_linux.go

```diff
@@ -137,6 +137,103 @@ func (self *CpuList) Get() error {
 
 	return err
 }
+func (self *NetProtoV6Stats) Get() error {
+	return readFile(Procd+"/net/snmp6", func(line string) bool {
+		fields := strings.Fields(line)
+		if len(fields) != 2 {
+			return true
+		}
+		switch fields[0] {
+		case "Ip6InReceives":
+			self.IP.InReceives, _ = strtoull(fields[1])
+		case "Ip6InAddrErrors":
+			self.IP.InAddrErrors, _ = strtoull(fields[1])
+		case "Ip6OutForwDatagrams":
+			self.IP.ForwDatagrams, _ = strtoull(fields[1])
+		case "Ip6InDelivers":
+			self.IP.InDelivers, _ = strtoull(fields[1])
+		case "Ip6OutRequests":
+			self.IP.OutRequests, _ = strtoull(fields[1])
+		case "Ip6InDiscards":
+			self.IP.InDiscards, _ = strtoull(fields[1])
+		case "Icmp6InMsgs":
+			self.ICMP.InMsgs, _ = strtoull(fields[1])
+		case "Icmp6InErrors":
+			self.ICMP.InErrors, _ = strtoull(fields[1])
+		case "Icmp6OutDestUnreachs":
```

Comment:
should this be `Icmp6InDestUnreachs`?  Do you also want to record `Icmp6OutDestUnreachs`?